spec/models/migration_engine_spec.rb

```diff
@@ -280,7 +280,7 @@
         it 'raises error with help message' do
           expect { migrations }.to raise_error do |error|
             expect(error).to be_a(MigrationEngine::MigrationEngineError)
-            expect(error.data).to eq(sle12ltss.friendly_name)
+            expect(error.data).to include(sle12ltss.friendly_name)
```

Comment:
I would add additionally to check the context as well but this is nitpicking here:
```suggestion
            expect(error.message).to match(/There are activated extensions\/modules on this system which cannot be migrated/)
            expect(error.data).to include(sle12ltss.friendly_name)
```